frameworks/helloworld/universe/config.json

```diff
@@ -1,158 +1,158 @@
 {
-  "type":"object",
-    "properties":{
```

Comment:
For the record, the `"properties"` element was incorrectly indented here. Hence the large looking diff.